source/manual/github-unavailable.html.md

```diff
@@ -6,123 +6,118 @@ layout: manual_layout
 parent: "/manual.html"
 ---
 
-## Public GitHub (application code)
+Many of the git repositories which make up GOV.UK are hosted on public GitHub. If GitHub becomes unavailable, our Jenkins deployment pipeline breaks. This is because:
 
-Many of the git repositories which make up GOV.UK are hosted on public GitHub. We may need to deploy changes at any time, and GitHub is a Software as a Service (SaaS) product which is not guaranteed to be available.
+* Jenkins uses the [GitHub Authentication plugin](https://plugins.jenkins.io/github-oauth/) to manage access: [only people in certain GitHub groups can access Jenkins](/manual/access-jenkins.html). If GitHub goes down, nobody can log into Jenkins
+* The [`Deploy_App` Jenkins job][Deploy_App]  uses GitHub by default when it clones repositories ready for deploying. If GitHub goes down, it can't clone the repository
 
-If GitHub is unavailable, we lose:
+The workaround is to disable Jenkins security, then configure the `Deploy_App` job to clone from AWS CodeCommit instead.
 
-* Access to our primary code repository
-* The ability to authenticate with Jenkins, as it makes use of GitHub groups
+## Disable Jenkins Security
 
-We [mirror all GitHub repositories](repository-mirroring.html) tagged with `govuk` to AWS CodeCommit every 2 hours. In the event of GitHub being down, we can deploy from AWS CodeCommit repos.
-
-### Deploying from AWS CodeCommit
-
-Use the normal deployment Jenkins job but check the box to deploy from AWS CodeCommit.
+If GitHub.com is down, we will not be able to log in to Jenkins.
+In this scenario, Jenkins security should be disabled to enable deployment:
 
-#### Making changes to code in AWS CodeCommit before deployment
+1. SSH to the Jenkins deploy instance: `gds govuk connect -e production ssh aws/jenkins`
+1. Disable Puppet: `govuk_puppet -r "Emergency Jenkins deploy" --disable`
+1. Edit the Jenkins configuration file: `sudo vim /var/lib/jenkins/config.xml`
+1. Replace `<useSecurity>true</useSecurity>` with `<useSecurity>false</useSecurity>` and save
+1. Restart Jenkins: `sudo service jenkins restart`
 
-1. In the root of the local repo, run the following commands to install the AWS
-   credential helper and add CodeCommit as a remote:
+Once security is disabled, anyone on GDS-trusted IPs will be able to invoke any Jenkins job, including `Deploy_App`. Do not leave security disabled for any longer than necessary.
 
-   ```
-   git remote add aws https://git-codecommit.eu-west-2.amazonaws.com/v1/repos/<app>
-   ```
+See the [Jenkins documentation](https://www.jenkins.io/doc/book/security/securing-jenkins/) for further details.
 
-1. Fetch the AWS upstream by running `git fetch aws`.
-   You'll need to provide a username and password, which you can retrieve from govuk-secrets
-   (pass/2ndline/govuk-repo-mirror/https_git_credentials.gpg).
```

Comment:
I've asked @beccapearce to try these instructions out to see if that's a viable workaround for the Mac issue 🤞